src/clj/harja/kyselyt/laskut.sql

```diff
@@ -12,6 +12,50 @@ WHERE l.urakka = :urakka
   AND l.erapaiva BETWEEN :alkupvm ::DATE AND :loppupvm ::DATE
   AND l.poistettu IS NOT TRUE;
 
+-- name: hae-johto-ja-hallintokorvaus-raporttiin-aikavalilla
+-- select * from
+
+-- name: hae-urakan-hj-kulut-raporttiin-aikavalilla
```

Comment:
Tulisiko hausta simppelimpi, jos käyttäisi left joinia?
Tyyliin näin:
select 
CASE koodi THEN koodi.nimi 
CASE ryhmä THEN ryhmä.nimi
END AS nimi, 
järkkä, 
ryhmä, 
summa
FROM
kt
left join instanssi ON instanssi.id = kt.instanssi
left join koodi ON koodi.id = kt.koodi AND koodi.nimi = 'Hoidonjohtopalkkio'
left join ryhmä ON ryhmä.id = kt.ryhmä AND ryhmä = yksiloiva_tunniste
WHERE
aikataulut täsmää